src/ImageSharp/Formats/Webp/WebpAnimationEncoder.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Six Labors.
+// Licensed under the Six Labors Split License.
+
+namespace SixLabors.ImageSharp.Formats.Webp;
+
+/// <summary>
+/// Encoder for animated webp images.
+/// </summary>
+public class WebpAnimationEncoder
+{
+    // 可能不需要这个屌东西
```

Comment:
Please write comments in English -- I'd like to be able to read them 😉 (w/o translator)

Reply:
I'm sorry! I was going to remove this class, but forgot.